go/consensus/tendermint/apps/staking/state/state_test.go

```diff
@@ -14,9 +14,14 @@ import (
 	memorySigner "github.com/oasislabs/oasis-core/go/common/crypto/signature/signers/memory"
 	"github.com/oasislabs/oasis-core/go/common/quantity"
 	"github.com/oasislabs/oasis-core/go/consensus/tendermint/abci"
+	epochtime "github.com/oasislabs/oasis-core/go/epochtime/api"
 	staking "github.com/oasislabs/oasis-core/go/staking/api"
 )
 
+const (
```

Comment:
Minor nitpick, can just have this as one line, and the constant definition should probably be in `TestDelegationQueries`

Reply:
yeah, done